test/bootstrap/compute_security_policies.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+gcloud services enable compute.googleapis.com
```

Comment:
You don't actually need to enable it since we have it enabled already in other scripts.

Reply:
Makes sense. I have commented it. Moreover, have also added some code to create a security policy.